src/Refactoring-UI/ReExtractMethodDriver.class.st

```diff
@@ -14,6 +14,34 @@ Class {
 	#tag : 'Drivers'
 }
 
+{ #category : 'displaying' }
+ReExtractMethodDriver class >> refactoringClass [
+
+	^ ReCompositeExtractMethodRefactoring
+]
+
+{ #category : 'execution' }
+ReExtractMethodDriver >> breakingChoices [
+
+	| items |
+	items := OrderedCollection new.
+	"
+	The following choices will be commited in short...
+	items add: (ReRenameMethodChoice new driver: self).
+	items add: (ReRenameAndBrowseMethodOverridesChoice new driver: self)."
+	items add: (ReBrowseMethodOverridesChoice new driver: self).
+	^ items
+]
+
+{ #category : 'actions' }
+ReExtractMethodDriver >> browseOverrides [
+
+	| conditions overrides |
+	conditions := refactoring failedBreakingChangePreconditions.
```

Comment:
I have done this because I know beforehand that the refactoring breaking change preconditions are all related to overrides. 

However, this doesn't look good because I shouldn't need to know this fact from the driver. 

The same happens if I store the refactoring breaking changes into IVs in the driver.

@balsa-sarenac what do you think? Maybe I am missing something important here